src/otx/algorithms/classification/adapters/mmcls/models/classifiers/custom_image_classifier.py

```diff
@@ -300,6 +304,57 @@ def extract_feat(self, img):
         ReciproCAMHook,
     )
 
+    def _extract_vit_feat(model, x):
```

Comment:
Could you update or add unit test for this?

Reply:
Thanks, added